### PR TITLE
Adds DangerJS for custom coding rules

### PR DESCRIPTION
**Pull request by cloudify**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -44,4 +44,7 @@ jobs:
       - run: '[ -z "$CC_TEST_REPORTER_ID" ] || ./cc-test-reporter before-build'
       - run: yarn test
       - run: '[ -z "$CC_TEST_REPORTER_ID" ] || (./cc-test-reporter after-build -t lcov < ./coverage/lcov.info)'
-      - run: 'bash <(curl -s https://codecov.io/bash)'
\ No newline at end of file
+      - run: 'bash <(curl -s https://codecov.io/bash)'
+
+      # Run DangerJS
+      - run: yarn danger ci
```
Comment (gunzip) [inline .circleci/config.yml]: where does this task come from ?

**Comment by cloudify** [comment]
@gunzip  `danger` is an executable, not a task

**Comment by gunzip** [comment]
yes, I didn't know that yarn can run locally installed bin without a package.json task =)

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cloudify** [comment]
yes it's a neat feature! :)

**Pull request merged**